colours/src/App.css

```
@@ -31,4 +31,8 @@
 
 .lime {
   color: lime
+}
+
+.teal {
+  color: teal;
 }
```

```suggestion

.teal {
  color: #f68390;
}
```

Lets use this instead